core/Test/Tasty/Ingredients/ConsoleReporter.hs

```diff
@@ -131,11 +134,32 @@ buildTestOutput opts tree =
       level <- ask
 
       let
+        postNamePadding = alignment - indentSize * level - stringWidth name
```

Comment:
Of course it's high school math, but.. this makes me need to think about operator precedence. Could you either do:

```suggestion
        postNamePadding = alignment - indentSize*level - stringWidth name
```

or:

```suggestion
        postNamePadding = alignment - (indentSize * level) - stringWidth name
```

Reply:
For real? 😆

Reply:
In Germany, this is _primary school_ math. 😆

Reply:
> For real? laughing

Yes, I wouldn't have posted it otherwise. Come on Andreas.

> but HLint is likely to suggest removing redundant brackets

Ah good point. Your suggestion removes all potential for confusion too, I like it.

Reply:
> FWIW I hate to think about precedence of operators in Haskell, but HLint is likely to suggest removing redundant brackets. I'd go for something like
> 
> ```haskell
> indentedNameWidth = indentSize * level + stringWidth name
> postNamePadding = alignment - paddedNameWidth
> ```

Was `indentedNameWidth` supposed to be `paddedNameWidth`?

Reply:
In general, I think the PR author has the right to ignore such nitpicks unless they are prescribed in a style guide published prior to the submission of the PR.  
It is here a call on the reviewers, not to review a PR to death.